lib/flowViewport/sceneIndex/fvpRenderIndexProxy.cpp

```diff
@@ -121,9 +125,26 @@ void RenderIndexProxy::RemoveSceneIndex(
     }
 }
 
-PXR_NS::HdSceneIndexBaseRefPtr RenderIndexProxy::GetMergingSceneIndex() const
+HdSceneIndexBaseRefPtr RenderIndexProxy::GetMergingSceneIndex() const
 {
     return _mergingSceneIndex;
 }
 
+HdRenderIndex* RenderIndexProxy::GetRenderIndex() const 
+{ 
+    return _renderIndex;
 }
+
+void RenderIndexProxy::GetRendererDisplayName(std::string& outRendererName) const 
```

Comment:
I usually prefer to pass the parameter to be filled as an out parameter instead of relying on the compiler to optimize the return value and avoid copies on the stack. But that's probably because I'm old school :-) 
But I guess for std::string they use std::move ? So I'd do it like you said.